addons/api/addon/models/storage-bucket.js

```diff
@@ -75,6 +77,14 @@ export default class StorageBucketModel extends GeneratedStorageBucketModel {
     return this.compositeType === TYPE_STORAGE_BUCKET_PLUGIN_AWS_S3;
   }
 
+  /**
+   * True if the storage bucket plugin type is MinIO
+   * @type {boolean}
+   */
+  get isMinio() {
```

Comment:
this is a nitpick - should we consolidate the usage here? how about MinIO?

Reply:
Sure! Nice catch! 😉